src-tauri/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gg"
-version = "0.27.0"
+version = "0.28.0"
```

Comment:
Not sure if we would use `0.28.1` for JJ `0.28.1`, or if the last number is reserved for changes in GG itself?